lang/cs/Org.Apache.REEF.Client/AzureBatch/Service/AzureBatchService.cs

```diff
@@ -111,14 +126,43 @@ public void CreateJob(string jobId, Uri resourceFile, string commandLine, string
                 // to the Job Manager Task (aka the Driver) as an environment variable.
                 // For more info, see
                 // https://docs.microsoft.com/en-us/dotnet/api/microsoft.azure.batch.cloudtask.authenticationtokensettings
-                AuthenticationTokenSettings = new AuthenticationTokenSettings() { Access = AccessScope.Job }
+                AuthenticationTokenSettings = new AuthenticationTokenSettings() { Access = AccessScope.Job },
+                ContainerSettings = CreateTaskContainerSettings(),
+                UserIdentity = new UserIdentity(autoUserSpecification: new AutoUserSpecification(elevationLevel: ElevationLevel.Admin)),
```

Comment:
This is only necessary for containers. Do we really want to elevate the level for Non-containers?